third_party/blink/common/web_preferences/web_preferences.cc

```diff
@@ -76,7 +76,13 @@ WebPreferences::WebPreferences()
       accelerated_filters_enabled(false),
       deferred_filters_enabled(false),
       container_culling_enabled(false),
+#if BUILDFLAG(IS_COBALT)
+      // Allow creating a ws: connection on a https: page to allow current
+      // testing set up. See b/377410179.
+      allow_running_insecure_content(true),
```

Comment:
Are we able to set this to true not via the upstream code, i.e. like https://source.chromium.org/chromium/chromium/src/+/main:components/content_settings/renderer/content_settings_agent_impl.cc;l=188?q=allow_running_insecure_content%20-f:out&ss=chromium%2Fchromium%2Fsrc ?